### DISCUSSION TITLE
It gives an error when I try to open an existing or new dashboard.

### DISCUSSION THREAD

**Discussion by melekalperatmaca** [eXpandFramework/eXpand#1057] [General]
I switched from version Devexpress 18.2.7.0 to version 22.1.12.0. My current Xafari version is 22.1.4011.0. My current Xpand version is 22.1.602.0. 
When I check the project, I see that there are no old dlls and modules left, they are all new. 
It gives an error when I try to open an existing or new dashboard. But the reports section works normally. Both existing reports and new reports are opened.
What could be the problem in the dashboard?
![image](https://github.com/eXpandFramework/eXpand/assets/74173139/1394953f-3555-4a5f-a311-8d32394f4831)
All dlls.
![image](https://github.com/eXpandFramework/eXpand/assets/74173139/d4ad17c6-786c-4f28-b64d-9fa27c7e284a)
PS: I also did what is written here : https://docs.devexpress.com/AspNet/403748/troubleshooting/installation-and-update-issues/type-x-exists-in-both-assemblies#error-description

Can you help me, please?
![image](https://github.com/eXpandFramework/eXpand/assets/74173139/a55dc861-6051-4836-82a5-4b59444282a7)

**Comment by apobekiaris** [comment]
the exception points to the following lines

>_modelRichEditEx = model.GetNode(ViewItemService.PropertyEditorControlMapName).Nodes().OfType<IModelRichEditEx>().First();
            ControlBindingProperty = _modelRichEditEx.ControlBindingProperty;

so for watever the reason the model was not extended, the most probable cause coud be the xpandWin module not installed

**Comment by melekalperatmaca** [comment]
thanks for your help solving this problem.
